lib/nostrum/struct/message.ex

```diff
@@ -250,6 +256,7 @@ defmodule Nostrum.Struct.Message do
       |> Map.update(:mentions, nil, &Util.cast(&1, {:list, {:struct, User}}))
       |> Map.update(:message_reference, nil, &Util.cast(&1, {:struct, Reference}))
       |> Map.update(:nonce, nil, &Util.cast(&1, Snowflake))
+      |> Map.update(:poll, nil, &Util.cast(&1, {:struct, Poll}))
```

Comment:
I'm not sure there is a noticeable increase here from the presence of empty keys?

I do understand where you are coming from, however surely since we are using atoms then we are not creating new values, and if there is no presence of keys then the empty fields.

As an experiment (and this is a very shoddy experiment), an additional empty key adds 2 words to the size of a map, that's irrelevant in my opinion.

```elixir
iex(8)> :erts_debug.size %{a: :b, c: "value"}
14
iex(9)> :erts_debug.size %{a: :b, c: "value", d: nil}
16
```

Reply:
But I'm just rambling for later, so we can resolve.